UBC-Meetable/code
# SuggestedFriendsScreen (onboarding process screen 4)

## Objective
This PR is to create the Suggested Friends screen of the onboarding process. 

Figma design: https://www.figma.com/file/MyozWvD25WGGMGeyIUTMcx/Untitled?node-id=0%3A1

Notes:

1) this screen is not currently hooked up to the onboarding process. I merely replaced the friends group screen with this one
for testing purposes. (**I switched it back before submitting this PR!**)

2)  This PR is mainly for functionality purposes. I will update the UI in a subsequent PR.

## Screenshots

What the screen looks like: 

![image](https://user-images.githubusercontent.com/46267622/141401801-47c34ffa-b8f9-4686-a055-5c030e2a5943.png)

When the user clicks **yes** followed by **done**, the _findFriendsOptionSelected_ updates to **true**

![findFriendsOptionSelected - true](https://user-images.githubusercontent.com/46267622/141402007-7bfdc026-5f03-4a35-be80-a342754841d9.jpg)

When the user clicks **no** followed by **done**, the _findFriendsOptionSelected_ updates to **false**

![findFriendsOptionSelected - false](https://user-images.githubusercontent.com/46267622/141402029-69ba169c-f68c-46db-a663-01288fb34d85.jpg)

## Testing instructions
1) In the _FriendGroupStackNavigator.tsx file_, replace the FriendsGroup component with the SuggestedFriendsScreen component. As shown in the first screenshot above, this will allow you to access the SuggestedFriendsScreen by clicking the middle button in the bottom navigator. **Please remember to switch the component back when you're done testing.**

2) run _npm run start_ in the src directory to run the app locally

3) run _amplify mock_ in a separate terminal window

4) Navigate to the SuggestedFriendsScreen on your app and click **yes** followed by **done**. Please note that this should log the user **id** on to the console on your terminal which is required for step 5.

5) On the amplify graphQL console, do a **getUser** query and use the **id** that was logged in step 4. You should see that the findFriendsOptionSelected field is true.

6) On the SuggestedFriendsScreen, click **no** followed by **done**.

7) On the amplify graphQL console, do a **getUser** query and use the id that was logged in step 4. You should see that the findFriendsOptionSelected field is now set to false.

## Discussion
For my SuggestedFriendsScreen's Stylesheet, I had to set marginLeft to 30 in order to center the page which I think is a bit weird. Anyone know a reason why it's not centered by default and if there's a more elegant way of centering it ? 

## Checklist
